template.rb

```diff
@@ -222,6 +219,10 @@ def add_esbuild_script
   end
 end
 
+def gem_exists?(name)
```

Comment:
Let's do an `add_gem` method that safely adds a gem.

Something like this:
```ruby
def add_gem(name, *options)
  gem(name, *options) unless gem_exists?(name)
end
```

Reply:
👍 Updated in a49f4f29f11cee913aa19c9f5b718a5b79a85506